tutorials/05_AnalysingModelOutputs.md

```diff
@@ -202,3 +198,27 @@ library(remind2)
 ```
 
 You can click on the index and search for interesting functions. All functions used to generate the reporting start with "report*.R".
+
+## 7. Visualizing Run Status und Summation Checks for Runs
+
+> **Note** This part assumes that you have access to the PIK HPC, have followed the instructions [in the REMIND group-internal Wiki](https://gitlab.pik-potsdam.de/rse/rsewiki/-/wikis/Cluster-Access) on how to access the PIK HPC and configure your cluster environment.
+
+### Run Status
+
+To get an overview of the status of your run, your can use the commandline tool `rs2` on the cluser (see help at `rs2 -h`). The tool provides information about runtime, convergence, number of iterations needed to converge, optimality etc.
```

Comment:
This is right now a placeholder for more info on rs2. As I don't have enough experience with this tutorial, someone else should provide more details here.